src/main.js

```diff
@@ -0,0 +1,171 @@
+import * as core from '@actions/core';
```

Comment:
You might want to add you own copyright here as well.

```suggestion
// SPDX-License-Identifier: Apache-2.0

import * as core from '@actions/core';
```